src/index.html

```diff
@@ -11,8 +11,47 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="/src/images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      />
+      <div class="card__title">
```

Comment:
It's better to use semantic tags where possible. For titles we can use headings tags `h1`, `h2`, ..., `h6`: https://www.w3schools.com/html/html_headings.asp